doc/api/deprecations.md

```diff
@@ -94,14 +94,6 @@ Within the [`child_process`][] module's `spawn()`, `fork()`, and `exec()`
 methods, the `options.customFds` option is deprecated. The `options.stdio`
 option should be used instead.
 
-<a id="DEP0007"></a>
-### DEP0007: cluster worker.suicide
-
-Type: Runtime
-
-Within the `cluster` module, the [`worker.suicide`][] property has been
-deprecated. Please use [`worker.exitedAfterDisconnect`][] instead.
```

Comment:
What I would recommend here is adding a short, non-emotional explanation why `worker.suicide` was renamed/removed.